v_4_4_0/files/k8s-resource/calico-all.yaml

```diff
@@ -501,6 +505,8 @@ rules:
     verbs:
       # Needed for clearing NodeNetworkUnavailable flag.
       - patch
+      # Calico stores some configuration information in node annotations.
```

Comment:
There are a few more RBAC changes but it seems related to TYPHA which we don't use IIRC